src/spring-cloud/azext_spring_cloud/tests/latest/test_asc_validator.py

```diff
@@ -230,13 +230,9 @@ def test_set_default_cidr_range(self):
             _set_default_cidr_range(
                 ['128.0.0.0/1', '0.0.0.0/2', '64.0.0.0/3', '96.0.0.0/4', '112.0.0.0/5', '120.0.0.0/6', '124.0.0.0/7',
                  '126.0.0.0/8'])
-            self.assertEqual(
-                'Cannot set "reserved-cidr-range" automatically.Please specify "--reserved-cidr-range" with 3 unused CIDR ranges in your network environment.',
-                str(context.exception))
-            _set_default_cidr_range(['0.0.0.0/1'])
```

Comment:
@yuwzho please help to verify whether this case is safe to remove. Currently it will not throw exception as its original expectation.

Reply:
LGTM